Makefile

```
@@ -795,6 +795,74 @@ packaging-macos-bundle:
+#   - pwsh, git, and the Go toolchain required by go.mod.
+#   - $(DIST_DIR) staged with the gateway zip + gateway-source-commit.txt
+#     from the macOS/Linux managed gateway step above.
+#   - Local git HEAD checked out at the same defenseclaw commit the gateway
+#     was built from; a mismatch is always rejected.
+#   - Cisco signing variables for production. Set
+#     WINDOWS_ENTERPRISE_SKIP_SIGNING=1 only for exact disposable
+#     certification scope.
+WINDOWS_ENTERPRISE_INSTALLER_OUT   ?= $(DIST_DIR)/windows-enterprise-installer
+WINDOWS_ENTERPRISE_INSTALLER_STATE ?= $(DIST_DIR)/windows-enterprise-installer-state
+WINDOWS_ENTERPRISE_SKIP_SIGNING    ?=
+packaging-windows-enterprise-installer:
```

checkmake (0.3.2)

[warning] Target body for "packaging-windows-enterprise-installer" exceeds allowed length of 5 lines (13).

(maxbodylength)